lvsfunc/comparison.py

```diff
@@ -536,6 +541,23 @@ def tvbd_diff(tv: vs.VideoNode, bd: vs.VideoNode,
     if not frames:
         raise ValueError("tvbd_diff: no differences found")
 
+    if write_log:
+        from itertools import groupby, count
+
+        def intervals(frames: list) -> list:
+            out = []
+            counter = count()
+
+            for key, group in groupby(frames, key=lambda x: x - next(counter)):
```

Comment:
you can use `_` for `key` as a placeholder to make linters happier since it's unused